TrackletAlgorithm/ProjectionRouterBuffer.h

```diff
@@ -141,8 +145,8 @@ template<int VMProjType, int AllProjectionType>
     return data_.range(kPRBufferTCIDMSB,kPRBufferTCIDLSB);
   }
   
-  SHIFT shift() const {
-    return data_.range(kPRBufferShiftMSB,kPRBufferShiftLSB);
+  PHIPROJBIN phiProjBin() const {
```

Comment:
@jasonfan393 For consistency with the other getters, can you start the name of this method with `get`?
```suggestion
  PHIPROJBIN getPhiProjBin() const {
```